backend/satellite_tools/spacewalk-debug

```diff
@@ -474,3 +474,26 @@ else
   rm -Rf echo $DIR
   echo "Debug dump created, stored in $TARBALL"
 fi
+
+# task schedules
+if [ -f /usr/bin/spacewalk-sql ] ; then
+    TASK_SCHEDULE_DEFAULT_SQL="""
+        SELECT id, job_label, bunch_id, org_id,
+          active_from, active_till, cron_expr,
+          data, created, modified
+        FROM rhntaskoschedule
+        WHERE cron_expr is not null or job_label like '%default';
+    """
+    echo "Extract default task schedules from the database - only the modified ones"
+    echo $TASK_SCHEDULE_DEFAULT_SQL | /usr/bin/spacewalk-sql --select-mode - > $DIR/task_schedule_default
```

Comment:
I think the query result could be put inside the task folder (`$DIR/tasko`) os export config, like the task logs are [1]

[1] https://github.com/uyuni-project/uyuni/blob/master/backend/satellite_tools/spacewalk-debug#L403-L405